r-package/grf/bindings/QuantileForestBindings.cpp

```diff
@@ -42,8 +44,18 @@ Rcpp::List quantile_train(std::vector<double> quantiles,
   data->set_outcome_index(outcome_index - 1);
 
   ForestOptions options(num_trees, ci_group_size, sample_fraction, mtry, min_node_size, honesty,
-      honesty_fraction, honesty_prune_leaves, alpha, imbalance_penalty, num_threads, seed, clusters, samples_per_cluster,
-      blocklength, blocknum);
+                        honesty_fraction, honesty_prune_leaves, alpha, imbalance_penalty, num_threads, seed, clusters, samples_per_cluster);
+  if(mbb){
+    options = ForestOptions{num_trees, ci_group_size, sample_fraction, mtry, min_node_size, honesty,
+                            honesty_fraction, honesty_prune_leaves, alpha, imbalance_penalty, num_threads, seed, clusters, samples_per_cluster,
+                            blocklength, blocknum};
+    if(nonoverlap){
+      options = ForestOptions{num_trees, ci_group_size, sample_fraction, mtry, min_node_size, honesty,
```

Comment:
We do not need to create another new constructor for ForestOptions. Just add one more arg to the current one.

Reply:
> We do not need to create another new constructor for ForestOptions. Just add one more arg to the current one.

Done.